### PR TITLE
Task/request state utils

### PR DESCRIPTION
**Pull request by giuliacampana**
utils for determining whether a request state is a specific type of request state.

### CODE REVIEW

File: src/utils/redux/requestIsFailure.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+ * @flow
+ */
+
+import { RequestStates } from 'redux-reqseq';
+import type { RequestState } from 'redux-reqseq';
+
+export default function requestIsFailure(requestState :?RequestState) :boolean {
```
Comment (UnsungHero97) [inline src/utils/redux/requestIsFailure.js:8]: We already know we're working with request states. Please rename to `isFailure`.

File: src/utils/redux/requestIsPending.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/*
+ * @flow
+ */
+
+import { RequestStates } from 'redux-reqseq';
+import type { RequestState } from 'redux-reqseq';
+
+export default function requestIsPending(requestState :?RequestState) :boolean {
```
Comment (UnsungHero97) [inline src/utils/redux/requestIsPending.js:8]: We already know we're working with request states. Please rename to `isPending`.

File: src/utils/redux/requestIsStandby.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/*
+ * @flow
+ */
+
+import { RequestStates } from 'redux-reqseq';
+import type { RequestState } from 'redux-reqseq';
+
+export default function requestIsStandby(requestState :?RequestState) :boolean {
```
Comment (UnsungHero97) [inline src/utils/redux/requestIsStandby.js:8]: We already know we're working with request states. Please rename to `isStandby`.

File: src/utils/redux/requestIsSuccess.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/*
+ * @flow
+ */
+
+import { RequestStates } from 'redux-reqseq';
+import type { RequestState } from 'redux-reqseq';
+
+export default function requestIsSuccess(requestState :?RequestState) :boolean {
```
Comment (UnsungHero97) [inline src/utils/redux/requestIsSuccess.js:8]: We already know we're working with request states. Please rename to `isSuccess`.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]
Don't forget to export via `src/utils/redux/index.js`.

File: src/utils/redux/requestIsFailure.js (hunk 1 above)
Comment (giuliacampana) [reply src/utils/redux/requestIsFailure.js:8]: oh ya, good point. i'll change all of these!

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**